com/AlterETM.py

```diff
@@ -161,7 +150,7 @@ def insert_modify_param(parser, cnn, stnlist, args):
                         parser.error('jump type == 1 but no relaxation parameter, please specify relaxation')
 
         elif ftype == 'q':
-            tpar.object      = 'periodic'
+            tpar.object = 'periodic'
             tpar.frequencies = [float(1/float(p)) for p in args.function_type[1:]]
```

Comment:
Break this line into 2? Python let's you do implicit line continuation over brackets like this:
```python
            tpar.frequencies = [float(1/float(p)) for p in
                                args.function_type[1:]]
```